modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -76,7 +83,14 @@ TensorMechanicsAction::TensorMechanicsAction(const InputParameters & params)
     _planar_formulation(getParam<MooseEnum>("planar_formulation").getEnum<PlanarFormulation>()),
     _out_of_plane_direction(
         getParam<MooseEnum>("out_of_plane_direction").getEnum<OutOfPlaneDirection>()),
-    _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : "")
+    _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : ""),
+    _cylindrical_axis_point1(isParamValid("cylindrical_axis_point1")
+                                 ? getParam<Point>("cylindrical_axis_point1")
+                                 : Point(0, 0, 0)),
+    _cylindrical_axis_point2(isParamValid("cylindrical_axis_point2")
+                                 ? getParam<Point>("cylindrical_axis_point2")
+                                 : Point(0, 1, 0)),
```

Comment:
I don't think we should have defaults for this. If the user asks for hoop, radial, or axial stress, we need checks to require them to be provided.

Reply:
There are over 50 input files which are using the default points and direction.  I can change them if needed